# Ensures length is properly typed (no-lenght)

This rule was created to prevent mistyping the length property.


## Rule Details

This rule aims to prevent writing lenght instead of length

Examples of **incorrect** code for this rule:

```js

someObj.lenght

```

Examples of **correct** code for this rule:

```js

someObj.length

```